tools/uninstall.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+# vim: tabstop=4 softtabstop=4 shiftwidth=4 noexpandtab
+###############################################################################
+# This is the Linux installation script for the King Phisher Client and
+# Server on supported distributions.
+#
+# Project Home Page: https://github.com/securestate/king-phisher/
+# Authors:
+#   Erik Daguerre
+#
+# Supported Linux Distributions:
```

Comment:
I don't even think this is necessary for the uninstallation script. I'd prefer to see it removed so it does not need to be updated when the install script is updated.